tests/ga/test_update.py

```diff
@@ -1247,8 +1247,8 @@ def iterator(*args, **kwargs):
 
         with patch('azurelinuxagent.ga.exthandlers.get_exthandlers_handler') as mock_handler:
             with patch('azurelinuxagent.ga.remoteaccess.get_remote_access_handler') as mock_ra_handler:
-                with patch('azurelinuxagent.ga.monitor.get_monitor_handler') as mock_monitor:
-                    with patch('azurelinuxagent.ga.env.get_env_handler') as mock_env:
+                with patch('azurelinuxagent.ga.update.get_monitor_handler') as mock_monitor:
+                    with patch('azurelinuxagent.ga.update.get_env_handler') as mock_env:
```

Comment:
Since I moved the `get_monitor_handler` and `get_monitor_handler` to the top of the file, these tests started breaking. I was debating if I should try to improve the tests (like we are supposed to do) or just hack it right now and decided to hack for now because -
- The PR is already too big, I did not want to add extra complexity to it. 
- The way these tests are written, they will fail if any new thread is added to the agent (which includes the ExtensionTelemetry thread). Currently the extension telemetry thread is disabled so its not affecting the test but once its enabled these tests would start failing. I will fix these tests for good when I finally enable that feature. That way it would be a separate more cleaner PR too.